site-config/lungmap/dev/index/projectsEntityConfig.ts

```diff
@@ -1,28 +1,47 @@
-import { EntityConfig } from "@databiosphere/findable-ui/lib/config/entities";
+import {
+  ComponentConfig,
+  EntityConfig,
+  SORT_DIRECTION,
+} from "@databiosphere/findable-ui/lib/config/entities";
+import { EXPLORE_MODE } from "@databiosphere/findable-ui/lib/hooks/useExploreMode";
+import {
+  HCA_DCP_CATEGORY_KEY,
+  HCA_DCP_CATEGORY_LABEL,
+} from "site-config/hca-dcp/category";
+import { ProjectsResponse } from "../../../../app/apis/azul/hca-dcp/common/responses";
+import {
+  getProjectId,
+  getTitle,
+} from "../../../../app/apis/azul/hca-dcp/common/utils";
+import * as C from "../../../../app/components";
+import { projectEdits as hcaProjectEdits } from "../../../../app/viewModelBuilders/azul/hca-dcp/common/projectMapper/projectEdits/projectEdits";
+import * as V from "../../../../app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders";
 import { mainColumn as hcaMetadataMainColumn } from "../../../hca-dcp/dev/detail/project/metadataMainColumn";
```

Comment:
Hi @jpaten is there is a reason why we haven't copied across `metadataMainColumn` like we have for `metadataSideColumn`?